scripts/scripts.js

```diff
@@ -145,6 +145,26 @@ const CONFIG = {
   atvCaptionsKey: 'bacom',
 };
 
+export const EVENT_LIBS = (() => {
+  const version = 'v1';
+  const { hostname, search } = window.location;
+
+  if (!(hostname.includes('.hlx.') || hostname.includes('.aem.') || hostname.includes('local'))) {
+    return `/event-libs/${version}`;
+  }
+
+  const branch = new URLSearchParams(search).get('eventlibs') || 'main';
+  if (branch === 'local') {
+    return `http://localhost:3868/event-libs/${version}`;
+  }
+
+  if (branch.includes('--')) {
+    return `https://${branch}.aem.live/event-libs/${version}`;
+  }
+
+  return `https://${branch}--event-libs--adobecom.aem.live/event-libs/${version}`;
+})();
+
```

Comment:
Do you think it'd be useful from a performance perspective to compact this into fewer lines to save file space?